src/server.js

```diff
@@ -149,10 +149,22 @@ function init(socket, client) {
 
 		socket.on("disconnect", function() {
 			client.clientDetach(socket.id);
+			sendConnectionInfo(client);
 		});
 		client.clientAttach(socket.id);
 
 		socket.on(
+			"remote-sign-out",
+			function(data) {
+				var connection_list = Object.keys(client.attachedClients);
+				for (var i = 0; i < connection_list.length; i++) {
+					if (data.socket_id === connection_list[i]) {
+						manager.sockets.of("/").connected[connection_list[i]].emit("sign-out", {});
```

Comment:
So, both, right? Emit disconnect event to the client, and after a timeout kill the connection on the server? Does this seem acceptable @xPaw?